docs/_static/vdr/v1.yaml

```diff
@@ -362,4 +378,44 @@ components:
         type:
           description: The type of the key.
           example: "JsonWebKey2020"
-          type: string
\ No newline at end of file
+          type: string
+    DIDCreateRequest:
+      properties:
+        controllers:
+          description: |
+            List of DIDs that can control the new DID Document. If selfControl = true and controllers is not empty.
+            The newly generated DID will be added to the list of controllers.
+          type: array
+          items:
+            type: string
+            description: DID according to Nuts specification
+            example: "did:nuts:128903fjgfslcnmgpe84"
+        assertionMethod:
+          type: boolean
+          description: indicates if the generated key pair can be used for assertions.
+          default: true
+        authentication:
+          type: boolean
+          description: indicates if the generated key pair can be used for authentication.
+          default: false
+        capabilityInvocation:
+          type: boolean
+          description: |
+            indicates if the generated key pair can be used for altering DID Documents.
+            In combination with selfControl = true, the key can be used to alter the new DID Document.
+            Defaults to true when not given.
+            default: true
+        capabilityDelegation:
+          type: boolean
+          description: indicates if the generated key pair can be used for capability delegations.
+          default: false
+        keyAgreement:
+          type: boolean
+          description: indicates if the generated key pair can be used for Key agreements.
+          default: false
+        selfControl:
+          type: boolean
+          description: wether the generated DID Document can be altered with its own capabilityInvocation key.
```

Comment:
what if `selfControl = false` and `capabilityInvocation = true`?

Reply:
Than that DID Document can control sub-documents.